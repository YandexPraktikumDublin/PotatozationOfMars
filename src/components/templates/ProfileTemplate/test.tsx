import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ProfileTemplate } from '.'

describe('<ProfileTemplate />', () => {
  it('should renders correct <ProfileTemplate />', () => {
    const wrapper = shallow(<ProfileTemplate />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
