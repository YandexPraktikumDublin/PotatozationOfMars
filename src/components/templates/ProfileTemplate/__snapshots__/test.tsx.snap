// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileTemplate /> should renders correct <ProfileTemplate /> 1`] = `
<div
  className="w-full max-w-2xl"
>
  <Window>
    <ProfileBoard />
  </Window>
</div>
`;
